urllib3/util/ssl_.py

```diff
@@ -17,6 +17,23 @@
 
 import errno
 import warnings
+import hmac
+
+if hasattr(hmac, 'compare_digest'):
```

Comment:
Off the top of my head, probably `urllib3/test/test_util.py` or something nearby.